### ISSUE TITLE
Add alt field to all image fields

### ISSUE THREAD

**Issue by tonifisler** [antistatique/antistatique.net#249]
Some are missing the `alt` field, like in the article content type.

**Comment by jminguely** [comment]
Is this still open ? I haven't been able to track a view without alt text. There is a lot of missing alt content on the images, but it's because the field are laid empty 😢
